examples/mp4info.rs

```diff
@@ -81,4 +109,15 @@ fn get_framerate(sample_counts: &Vec<u32>, duration: u32, timescale: u32) -> Str
     let sc = (sample_counts[0] as f64) * 1000.0;
     let ms = (duration as f64 / timescale as f64) * 1000.0;
     return format!("{:.2}", sc / ms.floor());
-}
\ No newline at end of file
+}
+
+fn format_creation_time(creation_time: u32) -> String {
+    // convert from MP4 epoch (1904-01-01) to Unix epoch (1970-01-01)
+    let time = if creation_time >= 2082844800 {
+        creation_time - 2082844800
+    } else {
+        creation_time
+    };
+    let dt = Utc.timestamp(time as i64, 0);
+    dt.format("%Y-%m-%d %H:%M:%S").to_string()
```

Comment:
Curious if it's possible to do this without the external dependency. I would like to keep the project as minimal as possible without adding any extra packages.

Reply:
None that I know of. Could just show the Unix epoch as a number and leave it at that. I don't care either way.